charmcraft/commands/store/registry.py

```diff
@@ -37,6 +37,10 @@
     MANIFEST_LISTS,
     MANIFEST_V2_MIMETYPE,
 }
+OCTET_STREAM_MIMETYPE = "application/octet-stream"
+
+# downloads and uploads happen in chunks
```

Comment:
ah, maybe the code comment needs to happen here :-)

Reply:
Improved this in the other PR